src/OpenTelemetry.AutoInstrumentation/Configurations/EnvironmentConfigurationMetricHelper.cs

```diff
@@ -103,8 +104,15 @@ public static MeterProviderBuilder AddAspNetCoreInstrumentation(MeterProviderBui
         [MethodImpl(MethodImplOptions.NoInlining)]
         public static MeterProviderBuilder AddHttpClientInstrumentation(MeterProviderBuilder builder, LazyInstrumentationLoader lazyInstrumentationLoader)
         {
-            DelayedInitialization.Metrics.AddHttpClient(lazyInstrumentationLoader);
+#if NET6_0_OR_GREATER
```

Comment:
This is little confusing as the project has only .NET6.0. Leaving a comment would help.

Reply:
Project is targeted also for .NET Framework 4.6.2.

I have added comment in f7eeae793ffcf06bb360815057fccb62b67885da. I am gonna merge it to unblock further work on .NET8 release. Please let me know, even after merge, if it is more clear now.

Reply:
I understand it. When someone else reviews `NET6_0_OR_GREATER` maybe misleading as it got only .NET 6.0, part from net462. In reality we are compiling only NET6 bits here. This is not a blocker, if you want to merge please proceed.